src/main/webapp/app/overview/course-conversations/dialogs/conversation-detail-dialog/conversation-detail-dialog.component.ts

```diff
@@ -34,13 +45,22 @@ export class ConversationDetailDialogComponent extends AbstractDialogComponent {
     @Input() selectedTab: ConversationDetailTabs = ConversationDetailTabs.MEMBERS;
 
     isInitialized = false;
+    isOneToOneChat = false;
+    otherUser?: ConversationUserDTO;
     readonly faPeopleGroup = faPeopleGroup;
+    readonly userNameClicked = output<number>();
 
     initialize() {
         super.initialize(['course', 'activeConversation', 'selectedTab']);
+        if (this.activeConversation) {
+            const conversation = getAsOneToOneChatDTO(this.activeConversation);
+            if (conversation) {
+                this.isOneToOneChat = true;
+                this.otherUser = conversation.members?.find((user) => !user.isRequestingUser);
+            }
+        }
```

Comment:
can it be that the `activeConversation` changes during the existence of the dialog? If yes, this might cause changes to not be re-rendered.

An alternative would be to rather create a computed-signal, like this (outside of `initialize`):
```typescript
private readonly activeConversation = computed(() => {
   const conversation = getAsOneToOneChatDTO(this.activeConversation);
      if (conversation) {
         this.isOneToOneChat = true;
         this.otherUser = conversation.members?.find((user) => !user.isRequestingUser);
      }
});
```

Reply:
I think using a computed signal here is unnecessary because the active conversation cannot change while the dialog is open. Since the dialog is not dismissed, switching to another conversation is not possible.